### ISSUE TITLE
Immovable rods not actually immovable

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#10852]
if you spawn 2 at the same time they don't stay at the same distance
literally unplayable

**Comment by Exxion** [comment]
There is no universal reference frame
Each of them is immovable from some reference frame or another

**Comment by 9600bauds** [comment]
Literally everything is not moving from some reference frame or the other

**Comment by clusterfack** [comment]
This is like the least sane thing to be triggered about

**Comment by Exxion** [comment]
Exactly, Bauds

**Comment by Ephx1** [comment]
Here, I know how to fix this
~~immovable~~ _Unstoppable_ rod

**Comment by PJB3005** [comment]
Uh

It can be interpreted that it's impossible to change the movement of the rod, not that's not moving. 

Newton's first law.

**Comment by Exxion** [comment]
That said, maybe they shouldn't be affected by timestop.

**Comment by Shadowmech88** [comment]
From an object's own frame of reference, unstoppable is essentially the same as immovable.

**Comment by Exxion** [comment]
Just rename it to the Unacceleratable Rod and we'll be golden

**Comment by NigglyWiggly** [comment]
No.
Immovable rod.
if it hits a singularity it creates an
U n s t o p p a b l e   F o r c e

**Comment by nuklearcellphoneg** [comment]
can we just remove them

**Comment by Ephx1** [comment]
@Shadowmech88 
Not necessarily. Immovable rods were something fixed in space that did not move, and only appeared to move due to whatever you were on moving instead. An unstoppable rod could be constantly moving, and in turn another unstoppable rod could appear at the same time being able to move at a different speed. Observing an immovable rod from another immovable rod would make the other one appear to be stationary, something that our current immovable rods show to be completely false.

**Comment by clusterfack** [comment]
No because theyre the best

**Comment by Shadowmech88** [comment]
Each immovable rod is only immovable from its own frame of reference.
From any other frame of reference, including that of a different immovable rod, it's an unstoppable rod.

**Comment by Ephx1** [comment]
Well no, because two immovable rods would view each other as unmoving. Two unstoppable rods could potentially cross paths, and yes while to itself it seems unmoving in this particular case we're viewing from outside of its view, exactly what bauds was mentioning, immovable rods are not immovable as two may cross paths, they're a reference to something that never moved from some D&D campaign.

**Comment by ConkerDev** [comment]
What happens when an unstoppable rod meats an unaccelerable rod

**Comment by clusterfack** [comment]
The ultimate question really

**Comment by ghost** [comment]
Immovable Rods are supposed to move at a constant speed along an unchangeable path, nothing can get in its way and stay in one piece

As was suggested a long while ago, it's more of a physics joke turned into a bus item

**Comment by ihadtoregisterforthis** [comment]
It's a reference to DND where there's an _almost_ immovable rod that can be left stationary to a relative point, even suspended mid air. A /tg/ thread tell the tale of THAT GUY who decided that the immovable rods stay still universally and as such they would devastate everything on a direction because the speed of the Earth.

**Comment by clusterfack** [comment]
No it's not a reference to DND, it's not a reference to WoW, seriously people look shit up first

https://en.wikipedia.org/wiki/Irresistible_force_paradox

**Comment by ihadtoregisterforthis** [comment]
http://suptg.thisisnotatrueending.com/archive/1709686/

**Comment by clusterfack** [comment]
How in the world do you think that is older than anything?

**Comment by ihadtoregisterforthis** [comment]
The universally fixed rod and _CLANG "what the fuck was that!?"_  both seem to come from there.
